### DISCUSSION TITLE
新的部署机制

### DISCUSSION THREAD

**Discussion by yyx990803** [vitejs/docs-cn#265] [Announcements]
/cc 参与翻译的各位：

大家也知道 Netlify 在国内很慢，最近经过一些调研之后，发现给国内加速的理想方案是 Gitee 免费提供的 Gitee Pages Pro。

测试地址：https://staging-cn.vitejs.dev/

这个方案目前看下来在国内各地访问速度都很不错，更关键的是不需要备案，可以继续使用 cn.vitejs.dev 的域名。

我配置了一个 GitHub Actions 自动把 `main` 的更新 push 到 Gitee: https://github.com/vitejs/docs-cn/commit/8826dc0eb26e9f9b43f98bec64e426471e508de1 - 理论上对这里的翻译工作流程不会有任何影响，只是知会大家一下（Netlify 的 build 切完之后会停掉）

**Comment by Otto-J** [comment]
这个证书域名过期了 https://staging-cn.vitejs.dev/
